### PR TITLE
Make management port configurable

### PR DESCRIPTION
**Pull request by niruhan**
## Purpose
$subject

### CODE REVIEW

File: integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java [hunk 1]
```diff
@@ -144,7 +144,12 @@ public synchronized void startServerUsingCarbonHome(String carbonHome, Map<Strin
                 }
             }));
 
-            managementPort = 9154 + portOffset;
+            if (commandMap.get("managementPort") != null) {
+                managementPort = Integer.parseInt(commandMap.get("managementPort")) + portOffset;
+            } else {
+                managementPort = 9763 + portOffset;
```
Comment (Nirothipan) [inline integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java:150]: shall we make MI port as default ( 9154 ) .. then we don't need to edit MI , automation xml files

File: integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java [hunk 2]
```diff
@@ -144,7 +149,12 @@ public synchronized void startServerUsingCarbonHome(String carbonHome, Map<Strin
                 }
             }));
 
-            managementPort = 9154 + portOffset;
+            if (StringUtils.isNotEmpty(commandMap.get("managementPort"))) {
```
Comment (Nirothipan) [inline integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java:152]: better to retrieve both from system prop . then no need to edit automation xml to run tests with EI ..

Comment (niruhan) [reply integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java:152]: From where we will set the System.property in connectors? (Classes inside connector and integration base gets hit after server starts)

Comment (Nirothipan) [reply integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java:152]: then how is that handled in scriptName = System.getProperty("startupScript");

Comment (niruhan) [reply integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java:152]: it's resolving to null and reading the value from scriptName = commandMap.get("startupScript");. I can remove the scriptName = System.getProperty("startupScript") part.

**Pull request merged**